src/init.cpp

```diff
@@ -206,6 +209,9 @@ pika::program_options::options_description getOptionsDescription() {
   desc.add_options()(
       "dlaf:band-to-tridiag-1d-block-size-base", pika::program_options::value<SizeType>(),
       "The 1D block size for band_to_tridiagonal is computed as 1d_block_size_base / nb * nb. (The input matrix is distributed with a {nb x nb} block size.)");
+  desc.add_options()(
+      "dlaf:tridiag-rank1-nworkers", pika::program_options::value<std::size_t>(),
+      "Maximum number of threads to use for computing rank1 problem solution in tridiagonal solver algorithm.");
```

Comment:
Only for consistency with other descriptions: 
```suggestion
      "The maximum number of threads to use for computing rank1 problem solution in tridiagonal solver algorithm.");
```